ElectricalCircuitSimulator/SimulationEngine/src/Matrix.cpp

```diff
@@ -20,27 +19,44 @@ namespace SimulationEngine {
 
         m_iNumRows = iRows;
         m_iNumColumns = iColumns;
-        m_oMatrix = new double*[m_iNumRows];
+        m_pMatrix = new double*[m_iNumRows];
         for(iRow = 0; iRow < m_iNumRows; iRow++) {
-            m_oMatrix[iRow] = new double[m_iNumColumns];
+            m_pMatrix[iRow] = new double[m_iNumColumns];
         }
         clear();
     }
 
+    Matrix::Matrix(const Matrix& oMatrix) {
```

Comment:
I'll let you look at my previous comments and see how you'd adapt this constructor yourself. Initialization lists are part of it.

Reply:
Will do later in Matrix rework